admin_guide/metrics_guidlines.adoc

```diff
@@ -0,0 +1,77 @@
+[[metrics-guidelines]]
+= OpenShift Container Platform (OCP) Metrics Guidlines
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+OpenShift Container Platform Metrics enable OCP users and administrators to monitor
+pods in OCP installation and quickly get information about specific pod resource utilization
+and based on that decide is it necessary to give additional resources to specific application pods in order to
+avoid capacity and performance issues.
+
+== Metrics Guidelines
+This document gives guidlines for OCP metrics deployment. For specific OCP metrics configuration
+aspets refer to OCP documentation ( Link to OCP v3.4 Administation guide )
+
+=== OpenShift Container Platform Metrics Node Capacity Planning
+
+Running OCP metrics nodes is recommended on separate OCP nodes than application ones.
```

Comment:
I think, there is no specific "infra node" documentation, from installer point of view, node is marked as infra based in node label. Which nodes will be marked as infra nodes depends on user. This document has recommendation for infra nodes to have at least 8cpus/32GB of ram